### PR TITLE
Nothing works, but we compile

### PR DESCRIPTION
**Pull request by valeratrades**
No clue how to split PRs at this stage. If I do any more, it will become too dispersed, so looked like a reasonable cut-off. Currently tried to hard-code order execution in the hub, so ignore it; implemented protocol feeds orders as it should. Next will do transform on orders for binance and then actual exchange runtime implemtation for it

### CODE REVIEW

**Comment by dev-ardi** [comment]
For changes that are only moving a file prefer doing that in a single "move" commit because otherwise the diff is too big

File: src/main.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#[macro_use]
+extern crate lazy_static;
```
Comment (dev-ardi) [inline src/main.rs:2]: Prefer `std::sync`'s `Once, `OnceLock` for the next time

**Comment by dev-ardi** [comment]
Btw can't really review changes in commits that large

File: src/main.rs (hunk 1 above)
Comment (valeratrades) [reply src/main.rs:2]: Ah, I've already rid of of globals, this is an artifact

You were right :P, something was going wrong with them, and I've still no clue what

**Comment by valeratrades** [comment]
Ah, and if it's a "." - don't even bother opening; it's either very minor, either a complete and utter madness.

**Pull request merged**